perl/p5-pgtop/Portfile

```diff
@@ -25,6 +25,8 @@ if {${perl5.major} != ""} {
     depends_lib-append \
                     port:p${perl5.major}-dbd-pg \
                     port:p${perl5.major}-dbi \
+                    port:p${perl5.major}-json \
+                    port:p${perl5.major}-lwp-useragent-cached \
```

Comment:
The `revision` must be increased when you add library or runtime dependencies.

Reply:
Fixed